src/app/query/stacks/utils.ts

```diff
@@ -30,3 +41,28 @@ export function createStacksAnchoredConfig(basePath: string) {
     fetchApi,
   });
 }
+
+export function convertQueriedTxsToWalletBitcoinTransactions(txs: BitcoinTransaction[]) {
```

Comment:
API/type wise, I'd break this into two functions. One that turns it into the desired type, another to take an array and `.map` it. (What if I only want to convert a single instance into this format, it only accepts an array).

Also, it's better to avoid type casting here. It legitimately _is_ the right type, so you'd better use the fn return type syntax. 

```ts
function createBitcoinTxTypeWrapper(tx: BitcoinTransaction): WalletBitcoinTransaction {
  return {
    blockchain: WalletBlockchains.bitcoin,
    transaction: tx,
  }
}
export function convertQueriedTxsToWalletBitcoinTransactions(txs: BitcoinTransaction[]) {
  return tx.map(tx => createBitcoinTxTypeWrapper(tx));
}
```

Reply:
https://www.bytelimes.com/why-you-should-avoid-type-assertions-in-typescript/